tools/block-generator/scenarios/config.payment.full.yml

```diff
@@ -1,6 +1,6 @@
 name: "Pay (full)"
-genesis_accounts: 10
-genesis_account_balance: 1000000000000000
+genesis_accounts: 10000
+genesis_account_balance: 1000000000000
```

Comment:
modifying these values so that very scenario will have the same genesis and can reuse the db from previous tests.

Reply:
2 PR's from now, I'm refactoring the config options and setting some defaults when they aren't explicitly provided. Perhaps these should be set as well and the configs need not explicitly provide them. But we can worry about that when we get to that PR.